doc/driver.mld

```diff
@@ -300,14 +300,7 @@ let dep_libraries_core = [
 ];;
 
 let extra_deps = [
-    "base";
-    "core_kernel";
-    "bin_prot";
-    "sexplib";
-    "sexplib0";
-    "base_quickcheck";
-    "ppx_sexp_conv";
-    "ppx_hash";
+    "core";
```

Comment:
How were the previous "extra deps" chosen? If the reason was valid, why not keeping them, adding "core"?

Reply:
I think the intent of the previous list was to have a world that is as large as core but that is not core, because it failed to build at some point. I might be wrong.

Reply:
No, it was that core had some dependencies and some were vaguely interesting, so might as well build them too. Particularly anything that core/core_kernel might have linked to (so sexplib, bin_prot, etc).

Reply:
Alright, I added them back. Though I did not add more as that would make the list very large without adding interesting test cases.

Reply:
My thinking was mainly that certain modules take vastly longer than others, so including a new library either means it's very quick (and not interesting) or that some modules now take ages, which is very interesting, and when you fix it, becomes a useful test case :-)